src/RiakClient/Models/MapReduce/KeyFilters/And.cs

```diff
@@ -13,64 +15,90 @@
 // KIND, either express or implied.  See the License for the
 // specific language governing permissions and limitations
 // under the License.
-
-using Newtonsoft.Json;
-using System;
-using System.Collections.Generic;
-using System.IO;
-using System.Text;
+// </copyright>
 
 namespace RiakClient.Models.MapReduce.KeyFilters
 {
+    using System;
+    using System.Collections.Generic;
+    using System.IO;
+    using System.Text;
+    using Newtonsoft.Json;
```

Comment:
Personally, I like having `using` statements inside of the namespace for the reasons outlined here:

http://stackoverflow.com/a/151560

Resharper settings to implement it: http://stackoverflow.com/a/8163511